yarn.lock

```diff
@@ -464,14 +464,6 @@ babel-jest@^16.0.0:
     babel-plugin-istanbul "^2.0.0"
     babel-preset-jest "^16.0.0"
 
-babel-jest@^19.0.0:
```

Comment:
Not sure the deal with these changes, I think I updated my version of `yarn` locally and it generated this on a `yarn install`.

Reply:
I think I had the same thing, looking at those dips it's probably from removing danger, which makes it my fault in #746 - should be fine